src/test/java/com/salesforce/dynamodbv2/QueryTest.java

```diff
@@ -100,178 +106,134 @@ void queryWithKeyConditionsEq(TestArgument testArgument) {
 
     // test legacy calls (see {@code QueryMapper} for more on "legacy".
     /**
-     * Table has (hk, rkLow), (hk, rkMiddle), (hk, rkHigh); we ask for items that match hk and have rk > rkLow, so there
```

Comment:
instead of filling the table with three objects whose range keys are -1, 0, 1, it's an adjustable min and max (currently set at -1 and 10, respectively)—this was to allow for a paging test that uses a page size that isn't super small (i.e., 2 or 1)